emsdk/patches/0003-dylink-Fix-rpath-calculation-in-nested-dependencies.patch

```diff
@@ -0,0 +1,76 @@
+From c371fb11a1c5b557557b2681a9b6be4cdaa78644 Mon Sep 17 00:00:00 2001
+From: ryanking13 <def6488@gmail.com>
+Date: Fri, 30 May 2025 15:54:17 +0900
+Subject: [PATCH 1/1] [dylink] Fix rpath calculation in nested dependencies
+
+Upstream PR:
+* https://github.com/emscripten-core/emscripten/pull/24234
+* https://github.com/emscripten-core/emscripten/pull/24461
```

Comment:
Could we keep these two as separate patches since they are separate upstream PRs?

Reply:
Separated patches.